examples/plugins/hello-world-wasm-rust/Makefile

```
@@ -0,0 +1,80 @@
+
+check-rust: ## Check if Rust and WASM target are installed
+	@which cargo > /dev/null 2>&1 || (echo "$(COLOR_ERROR)Error: Rust/Cargo is not installed$(COLOR_RESET)"; \
+		echo "$(COLOR_INFO)Install Rust: https://rustup.rs/$(COLOR_RESET)"; \
+		exit 1)
+	@rustup target list --installed | grep -q $(TARGET) || (echo "$(COLOR_ERROR)Error: WASM target not installed$(COLOR_RESET)"; \
+		echo "$(COLOR_INFO)Install with: rustup target add $(TARGET)$(COLOR_RESET)"; \
+		exit 1)
+	@echo "$(COLOR_SUCCESS)✓ Rust found: $$(rustc --version)$(COLOR_RESET)"
+	@echo "$(COLOR_SUCCESS)✓ WASM target: $(TARGET)$(COLOR_RESET)"
+
+build: check-rust ## Build the WASM plugin
```

checkmake (0.2.2)

[warning] Target body for "build" exceeds allowed length of 5 (6).

(maxbodylength)